docs/dashboards/kpi-component.md

```diff
@@ -0,0 +1,137 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
```

Comment:
I think it's worth mentioning that can be represented by value and you can show sth on the chart.